maps/README.txt

```diff
@@ -0,0 +1,6 @@
+The goal of this project is to create a sample interactive ad creative for SIMID. 
+Our aim is to design a creative using the Google Maps API that provides users a 
+map of nearby locations where they could find the advertised product.
+
+The files in this directory include: 
```

Comment:
You don't need to actually list the files, just a description of in general what will go here.
I.e. something like `Contains files for SIMID creative that will populate the iframe`

Reply:
Got it! Fixed